sw/airborne/modules/computer_vision/viewvideo.c

```diff
@@ -48,13 +48,13 @@
 
 // Downsize factor for video stream
 #ifndef VIEWVIDEO_DOWNSIZE_FACTOR
-#define VIEWVIDEO_DOWNSIZE_FACTOR 4
+#define VIEWVIDEO_DOWNSIZE_FACTOR 1
```

Comment:
Makes everything more flyable... when using this module as it is in master on the ARDrone it hardly receives telemetry updates (such as from the joystick) as the whole wifi fills with video. IMO this downsize factor makes it way better.